library/alloc/benches/vec_deque_append.rs

```diff
@@ -5,6 +5,11 @@ const WARMUP_N: usize = 100;
 const BENCH_N: usize = 1000;
 
 fn main() {
+    if cfg!(miri) {
+        // Don't benchmark Miri...
+        // (Due to cargo quirks, this gets picked up by `x.py test library/alloc --no-doc`.)
```

Comment:
Is this a "quirk"? I thought Cargo deliberately ran benchmarks as tests.

Reply:
This benchmark *only* gets run when you add `--no-doc`. I'd call that a quirk.^^

It is related to https://github.com/rust-lang/cargo/issues/13668.